### ISSUE TITLE
Accessibility: Option to completely hide custom emojis

### ISSUE THREAD

**Issue by annando** [friendica/friendica#14421]
Custom emojis had been introduced by Mastodon and many other systems in the Fediverse now use them as well. They have got an accessibility problem since for a screenreader user they simply appear as some kind of picture in the middle of the text. To improve the accessibility it is better to completely hide them. This should be a setting.
